samples/bluetooth/direction_finding_central/README.rst

```diff
@@ -85,6 +85,10 @@ This also means that, for example, when using four GPIOs, the pattern count cann
 
 If the number of switch-sample periods is greater than the number of stored switching patterns, then the radio loops back to the first pattern.
 
+Maximum length of antenna switch pattern is limited by :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.
```

Comment:
The form "The maximum .. is limited by .." is redundant. I suggest removing "maximum":
```suggestion
The length of the antenna switch pattern is limited by :kconfig:`CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN`.
```

Reply:
Corrected in all README.rst